pkg/daemon/update.go

```diff
@@ -461,6 +464,8 @@ func calculatePostConfigChangeActionFromFileDiffs(diffFileSet []string) (actions
 			actions = []string{postConfigChangeActionReloadCrio}
 		} else if ctrlcommon.InSlice(path, filesPostConfigChangeActionRestartCrio) {
 			actions = []string{postConfigChangeActionRestartCrio}
+		} else if ctrlcommon.InSlice(filepath.Dir(path), dirsPostConfigChangeActionReloadCrio) {
```

Comment:
Hmm, for the initial implementation of [nodeDiruptionPolicy](https://github.com/openshift/api/pull/1764) we don't actually allow wildcarding of paths like this (so all of these will eventually get converted to an API object)

I guess since that's tech preview we can always update the API or MCO processing to allow that but just a note that these are technically conflicting